discord_bot/events.py

```diff
@@ -1,7 +1,12 @@
 from enum import Enum
 
 
-class Events (Enum):
+class NoValue(Enum):
```

Comment:
What's the purpose of removing the assigned value from the enumeration? This complicates the code without adding additional functionality, and even makes it harder to debug issues if you need to know the assigned value. I recommend removing this NoValue class since it provides us with...no value ;3